plaso/parsers/azure_applicationgatewayaccess.py

```diff
@@ -0,0 +1,201 @@
+  """Azure application gateway access log event data.
+
+  Attributes:
+    instanceId (str): Application Gateway instance that served the request.
+    clientIP (str): Originating IP for the request.
+    clientPort (int): Originating port for the request.
+    httpMethod (str): HTTP method used by the request.
+    originalRequestUriWithArgs (str): This field contains the original request URL
+    requestUri (str): URI of the received request.
+    requestQuery (str): Server-Routed: Back-end pool instance that was sent the request. X-AzureApplicationGateway-LOG-ID: Correlation ID used for the request. It can be used to troubleshoot traffic issues on the back-end servers. SERVER-STATUS: HTTP response code that Application Gateway received from the back end.
+    userAgent (str): User agent from the HTTP request header.
+    httpStatus (int): HTTP status code returned to the client from Application Gateway.
+    httpVersion (str): HTTP version of the request.
+    receivedBytes (int): Size of packet received, in bytes.
+    sentBytes (int): Size of packet sent, in bytes.
+    clientResponseTime (int): Length of time (in seconds) that it takes for the first byte of a client request to be processed and the first byte sent in the response to the client.
+    timeTaken (double): Length of time (in seconds) that it takes for the first byte of a client request to be processed and its last-byte sent in the response to the client. It's important to note that the Time-Taken field usually includes the time that the request and response packets are traveling over the network.
+    WAFEvaluationTime (str): Length of time (in seconds) that it takes for the request to be processed by the WAF.
+    WAFMode (str): Value can be either Detection or Prevention
+    transactionId (str): Unique identifier to correlate the request received from the client
+    sslEnabled (str): Whether communication to the back-end pools used TLS. Valid values are on and off.
+    sslCipher (str): Cipher suite being used for TLS communication (if TLS is enabled).
+    sslProtocol (str): SSL/TLS protocol being used (if TLS is enabled).
+    sslClientVerify (str):
+    sslClientCertificateFingerprint (str):
+    sslClientCertificateIssuerName (str):
+    serverRouted (str): The backend server that application gateway routes the request to.
+    serverStatus (str): HTTP status code of the backend server.
+    serverResponseLatency (str): Latency of the response (in seconds) from the backend server.
+    originalHost (str): This field contains the original request host name
+    host (str): Address listed in the host header of the request. If rewritten using header rewrite, this field contains the updated host name
+  """
+
+  DATA_TYPE = 'azure:applicationgatewayaccess:json'
+
+  def __init__(self):
+    """Initializes event data."""
+    super(AzureApplicationGatewayAccessEventData, self).__init__(data_type=self.DATA_TYPE)
+
+    self.instanceId = None
```

Comment:
Please stick with the style in the guide https://google.github.io/styleguide/pyguide.html#3164-guidelines-derived-from-guidos-recommendations